packages/rust_verifier/pubspec.yaml

```diff
@@ -0,0 +1,50 @@
+name: ndk_rust_verifier
+description: Nostr Development Kit - the most performant lib for all your nostr usecases
+version: 0.1.3
+homepage: https://github.com/relaystr/ndk
+
+environment:
+  sdk: ">=3.4.0 <4.0.0"
+  flutter: ">=1.17.0"
+
+platforms:
+  android:
+  ios:
+  linux:
+  macos:
+  web:
+  windows:
+
+dependencies:
+  flutter:
+    sdk: flutter
+  http: ^1.2.2
+  bip340: ^0.3.0
+  bech32: ^0.2.2
+  hex: ^0.2.0
+  web_socket_channel: ^3.0.1
+  crypto: ^3.0.5
+  collection: ^1.18.0
+  path_provider: ^2.1.4
+  convert: ^3.1.1
+  pointycastle: ^3.9.1
+  plugin_platform_interface: ^2.1.7
+  logger: ^2.4.0
+  rxdart: ^0.28.0
+  flutter_rust_bridge: 2.6.0
+  rust_lib_ndk: ^0.1.3
+  ndk:
```

Comment:
does this work when publishing a package? Is melos smart enouth to replace with the new deployed version when pushing?

Reply:
not sure, to be tested/researched